config/routes.rb

```diff
@@ -1,2 +1,4 @@
 Rails.application.routes.draw do
+  # Defines the root path route ("/")
```

Comment:
@GowthamK01 Can we remove these comments if this is not required

Reply:
@gomathi-mallow If we remove the comment, Rubocop shows empty block detected offence. Shall we remove the comments after adding routes in further tasks?